doc/api/deprecations.md

```diff
@@ -1050,6 +1050,16 @@ deprecated along with the undocumented aliases `crypto.prng()` and
 `crypto.rng()` in favor of [`crypto.randomBytes()`][] and will be removed in a
 future release.
 
+<a id="DEP00XX"></a>
+### DEP00XX: Legacy URL API
+
+Type: Documentation
+
+The [Legacy URL API][] is deprecated in favor of the newer standardized
+[WHATWG URL][] implementation. This includes uses of [`url.format()`][],
+[`url.parse()`][], [`url.resolve()`][], and the [legacy `urlObject`][].
```

Comment:
@jasnell Is it OK with you if I change the text to this?:

```markdown
The [Legacy URL API][] is deprecated.  This includes  [`url.format()`][],
[`url.parse()`][], [`url.resolve()`][], and the [legacy `urlObject`][].  Please
use the [WHATWG URL API][] instead.
```

(This will also require updating the link reference currently on line 1158 to say `WHATWG URL API` rather than `WHATWG URL`.)

Reply:
Whatever you'd like